modules/olm-catalogsource.adoc

```diff
@@ -32,6 +32,9 @@ metadata:
   generation: 1
   name: example-catalog <.>
   namespace: {global_ns} <.>
+  annotations:
+    olm.catalogImageTemplate: <.>
+      "quay.io/example-org/example-catalog:v{kube_major_version}.{kube_minor_version}.{kube_patch_version}"
```

Comment:
The same as above comments.

Reply:
Is it OK if this example in the general "Catalog source" section stays like this? I'd rather keep the user/org-name simpler in this section as `example-org` but I did want to include `olm.catalogImageTemplate` as an "Optional" item. I feel like I'd need to change the name down in `spec.image` to match otherwise?

Reply:
IMO, I think I'd prefer to keep most of the examples as simple as possible like what's currently here Alex. I think it makes sense for us to show users what they ideally should be doing rather than a complete example of all the possible values that could be set and how.

Reply:
OK, leaving the `example-org` in place here and reverting the [other instance](https://github.com/openshift/openshift-docs/pull/36199#discussion_r706988315) back as well.